### PR TITLE
v.multi2singlepart: a new addon to split multipart to singlepart polygons

### PR DESCRIPTION
**Pull request by ecodiv**
Simple addon to split multipart polygons to singlepart polygons. p.s. I created this because I couldn't find an obvious way to do this. I feel there should be, so please let me know if I missed it.

### CODE REVIEW

**Comment by ecodiv** [comment]
@wenzeslaus if we have an improved v.dissolve, I think it would be good to also have a function in core that does, sort of, the opposite, split multi features into single features. To me, this feels like something that should be standard available? I don't think there is such a function? I am not saying this script is the way to go, there might be better ways.

File: src/vector/v.multi2singlepart/v.multi2singlepart.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+To go from singlepart to multipart features, use the 
+<em>v.dissolve</em> function (see example below).
+
+
+<h2>EXAMPLES</h2>
+
+The example uses the layer <i>boundary_municp</i> from the North 
+Carolina dataset. You can download the sample data set from the <a 
+href="https://grass.osgeo.org/download/data/">grass gis website</a>
```
Comment (echoix) [inline src/vector/v.multi2singlepart/v.multi2singlepart.html]:
```suggestion
href="https://grass.osgeo.org/download/data/">GRASS GIS website</a>
```

File: src/vector/v.multi2singlepart/v.multi2singlepart.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+# -
+# %module
+# % description: Split multi-polygon features into single polygon features.
+# % keyword: vector
+# % keyword: geometry
+# %end
+
+# %option G_OPT_V_INPUT
+# % description: Input vector layer
+# % required : yes
```
Comment (echoix) [inline src/vector/v.multi2singlepart/v.multi2singlepart.py]: Does the parser work correctly with spaces after `required`?
```suggestion
# % required: yes
```

File: src/vector/v.multi2singlepart/v.multi2singlepart.py [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+# %end
+
+# %option G_OPT_V_OUTPUT
+# % description: Output vector layer
+# % required : yes
```
Comment (echoix) [inline src/vector/v.multi2singlepart/v.multi2singlepart.py]:
```suggestion
# % required: yes
```
Same thig, if it is needed by the parser

**Pull request merged**